### DISCUSSION TITLE
Phantom not opening any transaction window (no alert, no transaction opens), in Android (any device) but works on Windows/Firefox/Chrome

### DISCUSSION THREAD

**Discussion by paracetamol951** [phantom/docs#271] [Help & Support] [answered]
### Summary

When i open the app with chrome, on android, it opens phantom, but phantom stays on the home screen, no transaction is shown.
On Windows, it works (except i have the warning from blowfish).
As i could get any error, i watched android logcat to see phantom logs, i could only found one error, related to the feching of https://www.google.com/s2/favicons?domain=p-link.io which indeed resolves to a 404, because Google takes an eternity to provide it after our domain name change.
The problem is only present since approx 07/08/2024

### Example

https://p-link.io

### Steps to Reproduce

Use Android device, latest Chrome, latest Phantom, go to https://p-link.io connect wallet, use the pay button, it opens phantom, but nothing happens

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @paracetamol951,

Would you mind providing your deeplink for connecting and transaction signing request please?

**Comment by paracetamol951** [comment]
I am using https://www.npmjs.com/package/@solana/wallet-adapter-react and web3.js in a typescript react next project.
I am not using deeplinks, maybe the package does. The transaction only fails on Android, but not on a PC browser.
Is there a way to see the deeplink that the wallet adapter uses ?

**Comment by paracetamol951** [comment, marked as answer]
Damn, i finally found out why.
Using the web3.js adapter, i was calling : 
const signature = await wallet.sendTransaction(tx, connection);
But since update of phantom of august 2024, it was failling and not showing anything, because i had to call : 
var blockhashResponse = await connection.getLatestBlockhashAndContext();
const signature = await wallet.sendTransaction(tx, connection, { minContextSlot:blockhashResponse.context.slot });
The minContextSlot is now mandatory.
Thanks a lot for digging this problem, 7 days searching this as top priority, I was getting crazy.

**Reply by adamdelphantom** [reply]
Glad to hear you were able to get to the bottom of it!
